tests/TestData/Providers/StubsTestDataProviders.php

```diff
@@ -47,32 +45,151 @@ public static function stubClassProvider(): ?Generator
         }
     }
 
-    public static function coreStubMethodProvider(): ?Generator
+    public static function stubMethodProvider(): ?Generator
     {
-        foreach (PhpStormStubsSingleton::getPhpStormStubs()->getCoreClasses() as $className => $class) {
+        $classesAndInterfaces = PhpStormStubsSingleton::getPhpStormStubs()->getClasses() +
+            PhpStormStubsSingleton::getPhpStormStubs()->getInterfaces();
+        foreach ($classesAndInterfaces as $className => $class) {
             foreach ($class->methods as $methodName => $method) {
                 yield "method {$className}::{$methodName}" => [$methodName, $method];
             }
         }
+    }
+
+    public static function stubMethodParametersWithoutScalarTypeHintsProvider(): ?Generator
```

Comment:
Provider names should be renamed to:
`stubMethodParametersForScalarTypeHintsTestProvider`